sql-statements/sql-statement-create-table.md

```diff
@@ -70,170 +70,16 @@ This statement creates a new table in the currently selected database. See also
-
-```sql
-index_type:
-    USING {BTREE | HASH}
-```
-
-The `index_type` is currently only supported in syntax.
-
-```sql
-index_option:
-    KEY_BLOCK_SIZE [=] value
-  | index_type
-  | COMMENT 'string'
-  | {VISIBLE | INVISIBLE}
-```
-
-The `KEY_BLOCK_SIZE` in `index_option` is currently only supported in syntax.
-
-```sql
-reference_definition:
-    REFERENCES tbl_name (index_col_name,...)
-      [MATCH FULL | MATCH PARTIAL | MATCH SIMPLE]
-      [ON DELETE reference_option]
-      [ON UPDATE reference_option]
-
-reference_option:
-    RESTRICT | CASCADE | SET NULL | NO ACTION | SET DEFAULT
-```
-
-```sql
-table_options:
-    table_option [[,] table_option] ...
-
-table_option:
-    AUTO_INCREMENT [=] value
-  | AVG_ROW_LENGTH [=] value
-  | SHARD_ROW_ID_BITS [=] value
-  | PRE_SPLIT_REGIONS [=] value
-  | [DEFAULT] CHARACTER SET [=] charset_name
-  | CHECKSUM [=] {0 | 1}
-  | [DEFAULT] COLLATE [=] collation_name
-  | COMMENT [=] 'string'
-  | COMPRESSION [=] {'ZLIB'|'LZ4'|'NONE'}
-  | CONNECTION [=] 'connect_string'
-  | DELAY_KEY_WRITE [=] {0 | 1}
-  | ENGINE [=] engine_name
-  | KEY_BLOCK_SIZE [=] value
-  | MAX_ROWS [=] value
-  | MIN_ROWS [=] value
-  | ROW_FORMAT [=] {DEFAULT|DYNAMIC|FIXED|COMPRESSED|REDUNDANT|COMPACT}
-  | STATS_PERSISTENT [=] {DEFAULT|0|1}
-```
-
-The `table_option` currently only supports `AUTO_INCREMENT`, [`SHARD_ROW_ID_BITS`](/shard-row-id-bits.md), `PRE_SPLIT_REGIONS`, `CHARACTER SET`, `COLLATE`, and `COMMENT`, while the others are only supported in syntax. The clauses are separated by a comma `,`. See the following table for details:
+The following *table_options* are supported. Other options such as `AVG_ROW_LENGTH`, `CHECKSUM`, `COMPRESSION`, `CONNECTION`, `DELAY_KEY_WRITE`, `ENGINE`, `KEY_BLOCK_SIZE`, `MAX_ROWS`, `MIN_ROWS`, `ROW_FORMAT` and `STATS_PERSISTENT` are parsed but ignored.
 
 | Parameters | Description | Example |
 | ---------- | ---------- | ------- |
 | `AUTO_INCREMENT` | The initial value of the increment field | `AUTO_INCREMENT` = 5 |
-|`SHARD_ROW_ID_BITS`| To set the number of bits for the implicit `_tidb_rowid` shards |`SHARD_ROW_ID_BITS` = 4|
+| [`SHARD_ROW_ID_BITS`](/shard-row-id-bits.md)| To set the number of bits for the implicit `_tidb_rowid` shards |`SHARD_ROW_ID_BITS` = 4|
 |`PRE_SPLIT_REGIONS`| To pre-split `2^(PRE_SPLIT_REGIONS)` Regions when creating a table |`PRE_SPLIT_REGIONS` = 4|
 |`AUTO_ID_CACHE`| To set the auto ID cache size in a TiDB instance. By default, TiDB automatically changes this size according to allocation speed of auto ID |`AUTO_ID_CACHE` = 200|
 |`AUTO_RANDOM_BASE`| To set the initial incremental part value of auto_random. This option can be considered as a part of the internal interface. Users can ignore this parameter |`AUTO_RANDOM_BASE` = 0|
-| `CHARACTER SET` | To specify the string code for the table; currently only support UTF8MB4 | `CHARACTER SET` =  'utf8mb4' |
+| `CHARACTER SET` | To specify the character set for the table | `CHARACTER SET` =  'utf8mb4' |
```

Comment:
Maybe "character set" could be a link to https://docs.pingcap.com/tidb/dev/character-set-and-collation?